### DISCUSSION TITLE
Bump to v2.0.0a4?

### DISCUSSION THREAD

**Discussion by czechbol** [nextcord/nextcord#270] [Ideas]
Hey, i just wanted to ask if you guys could bump a version to the latest commit.

For some reason `channel.overwrites_for()` which I'm using does not work for me on the latest release. I get a `TypeError` when calling it (for a member).

<details>
  <summary>Traceback</summary>

```
Traceback (most recent call last):
  ...REMOVED USELESS PART...
    user_overw = channel.overwrites_for(member)
  File "/root/.local/lib/python3.10/site-packages/nextcord/abc.py", line 465, in overwrites_for
    return PermissionOverwrite.from_pair(allow, deny)
  File "/root/.local/lib/python3.10/site-packages/nextcord/permissions.py", line 714, in from_pair
    for key, value in allow:
  File "/root/.local/lib/python3.10/site-packages/nextcord/flags.py", line 134, in __iter__
    yield (name, self._has_flag(value.flag))
  File "/root/.local/lib/python3.10/site-packages/nextcord/flags.py", line 137, in _has_flag
    return (self.value & o) == o
TypeError: unsupported operand type(s) for &: 'int' and 'NoneType'
```

</details>

Now I tried to debug it, tested on the latest commit and IT WORKS. I looked through the diffs and I don't see any obvious fixes, so imma call this magic. I got the same behavior running in Python 3.8 as in 3.10.
So could you make a version bump for me? I don't really want to put my production bot (managing a server of over 2k members) directly onto git.

**Comment by TAG-Epic** [comment]
Hmmm yeah probably gonna get merged with a few other commits. Fyi you can freeze dependencies at a spesific commit

**Reply by czechbol** [reply]
true but it still is a git repo and not a package

**Reply by TAG-Epic** [reply]
And? What's the difference?

**Comment by VincentRPS** [comment]
I don’t think this will happen or if it does only on the github since we are trying to release a stable v2 of nextcord with the big 2 features, unless epic wants to do some weird release candidate stuff
